nodebuilder/share/share.go

```diff
@@ -43,8 +45,8 @@ type Module interface {
 // TODO(@distractedm1nd): These structs need to be autogenerated.
 type API struct {
 	Internal struct {
-		SharesAvailable           func(context.Context, *share.Root) error `perm:"public"`
-		ProbabilityOfAvailability func(context.Context) float64            `perm:"public"`
+		SharesAvailable           func(context.Context, *share.Root, ...peer.ID) error `perm:"public"`
```

Comment:
I'm not sure variadic params work with the RPC, but I will check for you. Worst case is it has to take an array